pkgs/tools/networking/linux-router/default.nix

```diff
@@ -0,0 +1,89 @@
+# SOURCE: https://github.com/NixOS/nixpkgs/pull/92787/files
```

Comment:
```suggestion
```

Reply:
Sorry, that was a note for myself when I started playing around with it :speak_no_evil: